examples/run_expt.py

```diff
@@ -8,6 +8,7 @@
 import sys
 from collections import defaultdict
 
+sys.path.insert(1, os.path.join(sys.path[0], '..'))

```

Comment:
You're probably already aware of this but just a quick reminder to take out this line before merging.